.gitignore

```diff
@@ -0,0 +1,2 @@
+# Goland
```

Comment:
Not sure if it makes sense to include this file in the sample.

Reply:
It makes as much sense as any other gitignore, in my view. I think people will be more annoyed if I or someone else accidentally commits my `.idea` directory.